lib/settings_base.py

```diff
@@ -1002,10 +1003,22 @@ def JINJA_CONFIG():
     # Other queues we prioritize below.
 
     # AMO Devhub.
+    'devhub.tasks.convert_purified': {'queue': 'devhub'},
+    'devhub.tasks.flag_binary': {'queue': 'devhub'},
+    'devhub.tasks.get_preview_sizes': {'queue': 'devhub'},
+    'devhub.tasks.handle_file_validation_result': {'queue': 'devhub'},
+    'devhub.tasks.handle_file_validation_result': {'queue': 'devhub'},
+    'devhub.tasks.handle_upload_validation_result': {'queue': 'devhub'},
+    'devhub.tasks.handle_upload_validation_result': {'queue': 'devhub'},
```

Comment:
`devhub.tasks.handle_upload_validation_result` and `devhub.tasks.handle_file_validation_result` duplicated too.